go/config/config.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019 The Grafeas Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"errors"
+	"io/ioutil"
+	"log"
+
+	fernet "github.com/fernet/fernet-go"
+	"gopkg.in/yaml.v2"
+)
+
+// File is the grafeas config file.
```

Comment:
File should be lowercase, comment should always match the method or type name.

Reply:
good point, cleaned up this file